ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/console/DefaultOutputCustomizer.java

```diff
@@ -185,32 +168,73 @@ private boolean endsWith(Path path, Path relativePath) {
         return true;
     }
 
-    /*
-     * Selects and shows the specified line of text in editor
+    /**
+     * Finds a file by its path, opens it in editor and sets the text selection and
+     * reveals according to the specified line and column numbers
+     * 
+     * @param file
+     * @param lineNumber
+     * @param columnNumber
      */
-    private void selectRange(EditorPartPresenter editor, int line) {
-        if (editor instanceof TextEditor) {
-            TextPosition startPosition = new TextPosition(line - 1, 0);
-            int lineOffsetStart = ((TextEditor) editor).getDocument().getLineStart(line - 1);
-            if (lineOffsetStart == -1) {
-                lineOffsetStart = 0;
-            }
+    static void openFileInEditorAndReveal(AppContext appContext, EditorAgent editorAgent, Path file,
```

Comment:
@benoitf @slemeur I've pushed a fixup commit that separates DefaultOutputCustomizer into AbstractOutputCustomizer (with some commonly used utility methods) and JavaOutputCustomizer. All the existing customizers are adopted to this change.
The similar thing is made for JUnit tests: DefaultOutputCustomizerTest is separated into BaseOutputCustomizerTest (contains commonly used utility methods) and JavaOutputCustomizerTest. All the existing output customizer tests are adopted to this change as well.